sites/fast-website/src/app/components/site-navigation/site-navigation.ts

```diff
@@ -0,0 +1,15 @@
+import { FASTElement, observable } from "@microsoft/fast-element";
+
+export class SiteNavigation extends FASTElement {
+    @observable
+    items: HTMLElement[];
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.items.forEach((item: any) => {
+            if (item instanceof HTMLElement) {
+                item.setAttribute("role", "listitem");
```

Comment:
@khamudom can you point to why we're passing a role of listitem to whatever is passed within this?

Reply:
Thanks for bringing this up, I should have made a note asking if adding a role was necessary or not. Since I was not using a ul and li in this nav list, I wanted to make sure each of the items is part of a list group.

Reply:
I'd consider using something like `repeat()` to return each slotted item into a list item. We'll want the items here (if `fast-anchor`) to present to AT as anchors. If we want to render the items into a list, I would try and do so in the template.

Reply:
Initially, I thought of using `repeat()`, with an anchor, but decided on slotted directive instead, to give the flexibility of being able to slot other components inside the nav.

Reply:
It's a matter of *where*. If you add it to `fast-anchor`, the control will not present to AT with a role of anchor. If you render each item within a list item, you get the list behavior and it doesn't clobber the role of the anchors passed into slots.

Reply:
sounds like it is not necessary for this scenario. I will remove it